config/Makefile.am

```diff
@@ -30,4 +31,5 @@ EXTRA_DIST = celery-beat-env.conf \
              celery-worker-env.conf \
              faf.conf.in \
              faf-web.conf.in \
+             faf-logging.conf \
```

Comment:
Just a teeny tiny suggestion: Move the added line one up so that the list is alphabetically sorted.

Reply:
I approve this message.